Makefile

```diff
@@ -119,7 +119,3 @@ runservices: my.env
 .PHONY: stop
 stop: my.env
 	${DC} stop
-
-.PHONY: dependencycheck
-dependencycheck: my.env
-	${DC} run --rm crontabber shell ./docker/run_dependency_checks.sh
```

Comment:
We can run this with `socorro-cmd depcheck` now, so we don't need this make rule. I updated the docs accordingly.